readme.md

```diff
@@ -134,6 +135,32 @@ Directories are recursive by default. Directories named `fixtures` and `helpers`
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+### The `--match` flag
```

Comment:
I think you should move this down below the `Only-tests` section. Maybe retitle to `Running test that match a pattern`. Should mention that this is more specific than `.only`.